[doc] Update pprzcenter and GCS presentation. (#3229)
paparazzi/paparazzi: doc/sphinx/source/getting_started/paparazzi_center_tour.rst

.. quickstart paparazzi_center_tour

.. _paparazzicenter:

=================
Paparazzi Center
=================

The Paparazzi Center is a graphical interface for configuring the drones and managing
the various paparazzi tools.

The drop-down on the top right corner select which set of aircrafts will be used.

Current aircraft is choosen via the top left drop-down.
The *id* on the left of the aircraft name is the **uniq** identifier of the aircraft among the set, between 0 and 255.

.. image:: pprz_center_conf.svg

Configuration tab
==================

The left side of this tab is dedicated to the configuration of the aircraft.

An aircraft is composed of an airframe, a flight plan, some settings, a radio and a telemetry configuration file. These concepts are explained later.

For now, remember that you can select other files, or edit the current file with the button.

.. note::

    The ``Edit`` button open the file with *gedit* by default. You can change it via ``File->Edit Settings->Text Editor``.

Once the aircraft is configured, the firmware can be build for a target. 

Some usual targets are:

> - **sim**: The basic fixedwing simulator written in OCaml without IMU simulation or any sensor models (noise, bias, etc) and mainly intended to validate your flight plan logic.

Reviewer: Not anymore, it is now a nps FDM in C.
```suggestion
- **sim**: This basic fixedwing simulator has no IMU simulation or any sensor models (noise, bias, etc) and mainly intended to validate your flight plan logic.
```

- **nps**: NPS is a more advanced rotorcraft and fixedwing simulator with sensor models and commonly uses JSBSim as FDM (Flight Dynamic Model).
- **ap**: This is for a *real* autopilot.

If the selected target is not a simulator, you can choose how you want to upload the
firmware to your autopilot, then flash it.

> Compilation output is shown in the console zone.

Reviewer: ```suggestion
Compilation output is shown in the console below.
```
